int32 joint_idx
float64 target_position
---
bool target_reached

